### DISCUSSION TITLE
Welcome to VinylDNS Discussions!

### DISCUSSION THREAD

**Discussion by remerle** [vinyldns/vinyldns#1055] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by rick-ar** [comment]
Hi !
I am starting with vinyldns and I am not able to understand how I connect the existing zones in my Powerdns with vinyldns in order to manage them.

can you help me?
